frontend/src/_styles/theme.scss

```diff
@@ -1879,7 +1881,7 @@ input:focus-visible {
   .alert {
     background: transparent;
     .text-muted { 
-      color: #c3c3c3!important;
```

Comment:
Making this change when theme is dark as, c3c3c3 was kindof not readable on dark bg, can revert if needed